src/GraphQLParser/ParserContext.Parse.cs

```diff
@@ -1327,7 +1331,6 @@ private GraphQLType ParseType()
                 return type;
             }
 
-            IncreaseDepth();
```

Comment:
In fact, this line should not be removed but I add needed calls in other places later.